apps/prairielearn/src/pages/administratorInstitutions/administratorInstitutions.html.ts

```diff
@@ -41,11 +41,22 @@ export function AdministratorInstitutions({
           <div id="institutions" class="card mb-4">
             <div class="card-header bg-primary text-white d-flex align-items-center">
               Institutions
+              <button
+                type="button"
+                class="btn btn-sm btn-light ml-auto"
+                data-toggle="modal"
+                data-target="#add-institution-modal"
+              >
+                <i class="fas fa-plus"></i>
+                <span class="d-none d-sm-inline">Add institution</span>
+              </button>
             </div>
+            ${addInstitutionModal({ csrf_token: resLocals.__csrf_token })}
```

Comment:
Nit: "components" like this should usually use `PascalCase`, so this should be `AddInstitutionModal`.

Reply:
This has been updated to `PascalCase` in commit [402fad6](https://github.com/PrairieLearn/PrairieLearn/pull/9045/commits/402fad6c5a45a0e8ff5187613d3563a1bbf7d8f0).